spec/integration/kibana/spec.yml

```
@@ -125,7 +125,17 @@ spec:
     name: "tags.yml"
     required: false
     $ref: "./tags.spec.yml"
-
+  - description: Folder containing Kibana SLO assets
+    type: folder
+    name: slo
+    required: false
+    contents:
+    - description: An SLO asset file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^{PACKAGE_NAME}-.+\.json$'
+      forbiddenPatterns:
+        - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden 
 versions:
```

We could add a patch here to ensure that this feature is only available in versions of the spec implemented by Kibana. This would likely mean to add something like this:
```suggestion
versions:
  - before: 3.3.0
    patch:
      - op: remove
        path: "/contents/13" # remove SLO definitions
```

Or 3.2.1 if we decide to add it to the next patch version.